src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -115,6 +115,11 @@ def create_connectedk8s(cmd, client, resource_group_name, cluster_name, https_pr
     helm_version = check_helm_version(kube_config, kube_context)
     telemetry.add_extension_event('connectedk8s', {'Context.Default.AzureCLI.HelmVersion': helm_version})
 
+    # Check for faulty pre-release helm versions
+    if "3.3.0-rc" in helm_version:
+        telemetry.set_user_fault()
+        raise CLIError("The current helm version is not supported for azure-arc onboarding. Please upgrade the helm version and try again.")
```

Comment:
>helm version [](start = 111, length = 12)

Please upgrade to a stabel helm release may be as the message? or specify the version minimum needed?

Reply:
Changed the message to upgrade to stable version as 3.0.0 also works but 3.3.0-rc dosent as it was a pre-release version

Reply:
Dont see the update.

---
In reply to: [484773581](https://github.com/Azure/azure-cli-extensions/pull/2321#discussion_r484773581) [](ancestors = 484773581)